packages/ra-ui-materialui/src/layout/AppBar.tsx

```diff
@@ -179,7 +192,7 @@ export interface AppBarProps extends Omit<MuiAppBarProps, 'title' | 'classes'> {
     logout?: JSX.Element;
     open?: boolean;
     title?: string | JSX.Element;
-    userMenu?: JSX.Element;
+    userMenu?: JSX.Element | boolean;
```

Comment:
```suggestion
    userMenu?: JSX.Element | false;
```

Reply:
I think this should work

Reply:
True is allowed because it will display the default user menu.